pb_app/lib/submission_fom/EyeDropper.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_colorpicker/flutter_colorpicker.dart';
+import 'package:pb_app/models/submission_form_state.dart';
+import 'package:provider/provider.dart';
+
+class EyeDropper extends StatelessWidget {
+  const EyeDropper({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final submissionFormState = context.read<SubmissionFormState>();
+    return IconButton(
+      icon: const Icon(Icons.colorize),
+      onPressed: () {
+        showDialog(
+          context: context,
+          builder: (_) {
+            return Theme(
+              data: _adaptThemeBrightness(
+                Theme.of(context),
+                submissionFormState,
+              ),
+              child: _DialogWidget(submissionFormState: submissionFormState),
+            );
+          },
+        );
```

Comment:
can we move this to a `static Future<void> show(BuildContext context)` in _DialogWidget near line 57?

This would repeat the pattern started by `lib/modals.dart:L13`

Reply:
Is there any merit to returning the `Future` until there is a caller that actually awaits it?